docs/standards/CX-0135-CompanyCertificateManagement/assets/OpenAPI_Spec.yaml

```diff
@@ -0,0 +1,588 @@
+          const: "REJECTED"
+        requestErrors:
+          type: array
+          description: Listing all reasons why the requested certificate can not be provided
+          items:
+            $ref: '#/components/schemas/Error'
+        locationErrors:
+          type: array
+          description: Optional detailed reasons for the rejection for each location
+          items:
+            $ref: '#/components/schemas/LocationErrorCollection'
+
+
+    BpnlType:
+      type: string
+      description: "A business partner number for legal entities. The provided regular expression ensures that the BPNL is composed of prefix 'BPNL', 10 digits and two alphanumeric letters."
+      pattern: "^BPNL[A-Z0-9]{12}$"
+
+    BpnsType:
+      type: string
+      pattern: "^BPNS[A-Z0-9]{12}$"
+
+    BpnLocation:
+      type: string
+      description: The BPNA or BPNS of either an address or site location
+      pattern: "^BPN[A|S][A-Z0-9]{11}$"
+      examples:
+        - "BPNA000000000001"
+        - "BPNS000000000002"
+
+    CertificateType:
+      description: "Detailed entity of the certificate like IS09001:2015, IATF 16949:2015 or other, valid types are registered at BPN metadatacontroller"
+      type: object
+      required:
+        - certificateType
+      properties:
+        certificateType:
+          type: string
+          description: "Type of the certificate as defined on the document,valid types are registered at BPN metadatacontroller"
+        certificateVersion:
+          type: string
+          description: "Version of the certificate as defined on the document, usually the specific version of a certification standard"
+
+
+    CertificatePush:
+      type: object
+      required:
+        - header
+        - content
+      properties:
+        header:
+          $ref: '#/components/schemas/FeedbackUrlHeader'
+        content:
+          type: object
+          required:
+            - businessPartnerNumber
+            - type
+            - registrationNumber
+            - validFrom
+            - trustLevel
+            - document
+          properties:
+            businessPartnerNumber:
+              description: The BPNL of the certified legal entity
+              $ref: "#/components/schemas/BpnlType"
+            type:
+              description: Type of the certificate as defined on the document
+              $ref: "#/components/schemas/CertificateType"
+            registrationNumber:
+              type: string
+              description: Registration number of the certificate as defined on the certificate
+            areaOfApplication:
+              type: string
+              description: Details on which areas / application types a certificate is valid for a company and/or site.
+            enclosedSites:
+              type: array
+              description: "Additional sites covered by the certificate, which can be either the Business Partner Number Site (BPNS) or Business Partner Number Address (BPNA)"
```

Comment:
Why is it called site then? And in the other requests such attribute is called location. Very misleading